src/main/java/com/cloudant/client/org/lightcouch/internal/CouchDbUtil.java

```diff
@@ -137,10 +140,14 @@ public static int getAsInt(JsonObject j, String e) {
         try {
             Class<CouchDbUtil> clazz = CouchDbUtil.class;
             URL dirURL = clazz.getClassLoader().getResource(path);
+            //Null when getting resources outside of workspace
```

Comment:
Afaict the `listResources` is no longer used anywhere, so do we really need this?